Gemfile.lock

```diff
@@ -1,158 +0,0 @@
-      i18n (>= 1.6, < 2)
-      minitest (>= 5.1)
-      mutex_m
-      tzinfo (~> 2.0)
-    addressable (2.8.7)
-      public_suffix (>= 2.0.2, < 7.0)
-    async (2.12.1)
-      console (~> 1.25, >= 1.25.2)
-      fiber-annotation
-      io-event (~> 1.6, >= 1.6.5)
-    async-http (0.69.0)
-      async (>= 2.10.2)
-      async-pool (~> 0.7)
-      io-endpoint (~> 0.11)
-      io-stream (~> 0.4)
-      protocol-http (~> 0.26)
-      protocol-http1 (~> 0.19)
-      protocol-http2 (~> 0.18)
-      traces (>= 0.10)
-    async-http-faraday (0.14.0)
-      async-http (~> 0.42)
-      faraday
-    async-pool (0.7.0)
-      async (>= 1.25)
-    base64 (0.2.0)
-    bigdecimal (3.1.8)
-    concurrent-ruby (1.3.3)
-    connection_pool (2.4.1)
-    console (1.25.2)
-      fiber-annotation
-      fiber-local (~> 1.1)
-      json
-    drb (2.2.1)
-    erubi (1.9.0)
-    faraday (2.9.2)
-      faraday-net_http (>= 2.0, < 3.2)
-    faraday-http-cache (2.5.1)
-      faraday (>= 0.8)
-    faraday-net_http (3.1.0)
-      net-http
-    ffi (1.17.0)
-    ffi (1.17.0-aarch64-linux-gnu)
-    ffi (1.17.0-aarch64-linux-musl)
-    ffi (1.17.0-arm-linux-gnu)
-    ffi (1.17.0-arm-linux-musl)
-    ffi (1.17.0-arm64-darwin)
-    ffi (1.17.0-x86-linux-gnu)
-    ffi (1.17.0-x86-linux-musl)
-    ffi (1.17.0-x86_64-darwin)
-    ffi (1.17.0-x86_64-linux-gnu)
-    ffi (1.17.0-x86_64-linux-musl)
-    fiber-annotation (0.2.0)
-    fiber-local (1.1.0)
-      fiber-storage
-    fiber-storage (0.1.2)
-    github_changelog_generator (1.16.4)
-      activesupport
-      async (>= 1.25.0)
-      async-http-faraday
-      faraday-http-cache
-      multi_json
-      octokit (~> 4.6)
-      rainbow (>= 2.2.1)
-      rake (>= 10.0)
-    i18n (1.14.5)
-      concurrent-ruby (~> 1.0)
-    io-endpoint (0.11.0)
-    io-event (1.6.5)
-    io-stream (0.4.0)
-    json (2.7.2)
-    minitest (5.24.1)
-    multi_json (1.3.6)
-    mustermann (2.0.2)
-      ruby2_keywords (~> 0.0.1)
-    mutex_m (0.2.0)
-    net-http (0.4.1)
-      uri
-    octokit (4.25.1)
-      faraday (>= 1, < 3)
-      sawyer (~> 0.9)
-    protocol-hpack (1.4.3)
-    protocol-http (0.26.6)
-    protocol-http1 (0.19.1)
-      protocol-http (~> 0.22)
-    protocol-http2 (0.18.0)
-      protocol-hpack (~> 1.4)
-      protocol-http (~> 0.18)
-    public_suffix (6.0.0)
-    rack (2.2.9)
-    rack-protection (2.2.4)
-      rack
-    rainbow (3.1.1)
-    rake (13.2.1)
-    rb-fsevent (0.11.2)
-    rb-inotify (0.11.1)
-      ffi (~> 1.0)
-    ruby2_keywords (0.0.5)
-    sass (3.7.4)
-      sass-listen (~> 4.0.0)
-    sass-listen (4.0.0)
-      rb-fsevent (~> 0.9, >= 0.9.4)
-      rb-inotify (~> 0.9, >= 0.9.7)
-    sawyer (0.9.2)
-      addressable (>= 2.3.5)
-      faraday (>= 0.17.3, < 3)
-    sinatra (2.2.4)
-      mustermann (~> 2.0)
-      rack (~> 2.2)
-      rack-protection (= 2.2.4)
-      tilt (~> 2.0)
-    tilt (2.4.0)
-    traces (0.11.1)
-    tzinfo (2.0.6)
-      concurrent-ruby (~> 1.0)
-    uri (0.13.0)
-    webrick (1.8.1)
```

Comment:
We just received a notification saying that this version of webrick is vulnerable to _HTTP Request Smuggling_

Getting rid of Gemfile.lock allows consumers to not depend on us for updating our dependencies.